rpcserver.go

```diff
@@ -1113,6 +1113,18 @@ func (r *rpcServer) GetInfo(ctx context.Context,
 		activeChains[i] = chain.String()
 	}
 
+	// Determine if a public IP address was set on lnd and use it to set the uri.
+	var hostPort net.Addr
+	addrs := r.server.currentNodeAnn.Addresses
+	if len(addrs) != 0 {
+		hostPort = addrs[0]
```

Comment:
Since it can have more that one external IP, we can just return a list of all.

Reply:
This means we would end up having a list of URIs as well.

Reply:
That's probably useful :) Take a look at `lncli getnodeinfo`